batch-models/src/main/scala/org/ekstep/analytics/exhaust/OnDemandDruidExhaustJob.scala

```diff
@@ -100,6 +100,9 @@ object OnDemandDruidExhaustJob extends optional.Application with BaseReportsJob
       (df.columns).map(f1 => {
         df = df.withColumn(f1, when((col(f1) === "unknown") || (col(f1) === "<NULL>"), "Null").otherwise(col(f1)))
       })
+      if (sortDfColNames != null) {
```

Comment:
Instead of doing null check - use Option[List[String]] as type for sortDfColNames and do isDefined check